packages/admin/admin/src/helpers/RootSlotProps.ts

```diff
@@ -0,0 +1,13 @@
+import { Theme } from "@mui/material/styles";
+import { MUIStyledCommonProps, SxProps } from "@mui/system";
+import React from "react";
+
+type SlotProps<Component extends React.ElementType> = Partial<React.ComponentProps<Component>> & MUIStyledCommonProps<Theme>;
+
+export type RootSlotProps<SlotKeyToElementType extends Record<string, React.ElementType> = never> = {
```

Comment:
I'm not sure about the naming here. This is more than the root slot IMO. What about `ThemedComponentBaseProps`? And `SlotKeyToElementType` could be just `Slots`?

Reply:
Yes, that sounds better 👍🏼